docs/theming.md

```diff
@@ -185,9 +185,170 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement. This migration guide will cover the typical migration path from Themer usage to
+Theming extensions.
+
+In general, every component's `Themer` targets will gradually be replaced by a single `Theming`
+extension target. This includes:
+
+- ColorThemer
+- FontThemer
+- ShapeThemer
+- TypographyThemer
+- Component Themers (e.g. CardThemer)
+
+Some Theming extensions are still in Beta. To add a Beta Theming extension to your project you will
```

Comment:
Tip: Is this doc only for beta? Should we also explain how to add non-beta theming extensions?